test/containerized_test.py

```diff
@@ -16,13 +25,113 @@
 from . import http
 
 
-TICKET_SIZE = 16 * KiB
+log = logging.getLogger("test")
+
+FILE_SIZE = 16 * KiB
+
+SERVER_PORT = 8080
+CONTAINER_IMAGE = "localhost/ovirt-imageio:latest"
+
+
+class ContainerPath:
+
+    def __init__(self, file_name, folder):
+        self.file_name = file_name
+        self.folder = folder
+
+    @property
+    def path(self):
+        return os.path.join(self.folder, self.file_name)
+
+
+class TestImage:
+
+    container = ContainerPath(file_name="disk.raw", folder="/image")
+
+    def __init__(self, tmpdir):
+        self.path = self._create_image(tmpdir)
+
+    def _create_image(self, tmpdir):
+        image_path = tmpdir.mkdir("image")
+        image = testutil.create_tempfile(
+            image_path, name=self.container.file_name, size=FILE_SIZE)
+        os.chmod(image, stat.S_IROTH | stat.S_IWOTH)
+        return image
+
+
+class TestTicket:
+
+    container = ContainerPath(file_name="file.json", folder="/ticket")
+
+    def __init__(self, tmpdir, tmp_image):
+        self.id = str(uuid.uuid4())
+        self.path = self._create_ticket(tmpdir, tmp_image)
+
+    def _create_ticket(self, tmpdir, tmp_image):
+        ticket = testutil.create_ticket(
+            uuid=self.id,
+            size=FILE_SIZE,
+            url=f'file://{tmp_image.container.path}')
+        ticket_dir = tmpdir.mkdir("ticket")
+        ticket_path = ticket_dir.join(self.container.file_name)
+        ticket_path.write(json.dumps(ticket))
+        return str(ticket_path)
+
+
+@pytest.fixture
+def tmp_image(tmpdir):
+    return TestImage(tmpdir)
+
+
+@pytest.fixture
+def tmp_ticket(tmpdir, tmp_image):
+    return TestTicket(tmpdir, tmp_image)
+
+
+@pytest.fixture
+def srv(tmpdir, tmp_ticket, tmp_image):
+    cmd = ["podman", "run", "--rm", "--privileged", "-it"]
```

Comment:
Apparently it does not work. I was not sure if it would work, but I had hopes. I just had never tried before. That sucks, but I'm not sure what else can be done to test it in the pipelines.